model_files/THDMII/LesHouches.in.THDMII

```diff
@@ -16,7 +16,7 @@ Block FlexibleSUSY
    12   0                    # force output
    13   1                    # Top pole mass QCD corrections (0 = 1L, 1 = 2L, 2 = 3L)
    14   1.000000000e-11      # beta-function zero threshold
-   15   0                    # calculate observables (a_muon, ...)
+   15   1                    # calculate observables (a_muon, ...)
```

Comment:
I remember that @pathron had some objection to enabling calculation of observables by default. I can't remember why though.

Reply:
You are right, we should probably not calculate all observables by default. I've just reverted this change.